board.js

```diff
@@ -0,0 +1,66 @@
+import { flipCard } from './main'
+
+const boardArray = Array.from({ length: 20 })
+const gameBoard = document.querySelector('.main__board')
+const boardCell = document.createElement('div')
+const scoresBoard = document.querySelector('.main__scores')
+const scoresHeader = document.createElement('h1')
+const result = document.createElement('h1')
+const imgs = ['img/cat-black.jpg', 'img/cat-climbs.jpg', 'img/cat-flower.jpg', 'img/cat-grey.jpg', 'img/cat-grey-white.jpg', 'img/cat-licks.jpg', 'img/cat-sitting.jpg', 'img/cat-sneaky-black.jpg', 'img/cat-watch.jpg', 'img/cat-watch-2.jpg']
```

Comment:
to refactor

Reply:
I don't understand what should I refactor?